### PR TITLE
Fix: #7. Add tests

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

refactor __main__ adding a smoke test

## Note

A comparison test can be added in another PRs

### CODE REVIEW

File: tests/test_generate.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from fastapi_code_generator.__main__ import generate_code
+from pathlib import Path
+import pytest
+
+
+@pytest.mark.parametrize("oas_file", Path("tests/").glob("*.yaml"))
+def test_generate_simple(oas_file):
+    input_file = oas_file  # "tests" / Path('simple.yaml')
```
Comment (ioggstream) [inline tests/test_generate.py:8]:
```suggestion
    input_file = oas_file 
```

File: tests/test_generate.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    ret = generate_code(input_name=input_file.name, input_text=input_file.read_text(), output_dir="tmp" / oas_file.with_suffix(".app"),
```
Comment (ioggstream) [inline tests/test_generate.py:9]:
```suggestion
    ret = generate_code(input_name=oas_file.name, input_text=oas_file.read_text(), output_dir="tmp" / oas_file.with_suffix(".app"),
```

File: tests/test_generate.py (hunk 1 above)
Comment (ioggstream) [inline tests/test_generate.py:8]:
```suggestion
```

**Comment by koxudaxi** [comment]
This PR looks great :rocket:

**Comment by koxudaxi** [comment]
Thank you very much!!

**Pull request merged**